scout/dashboard.md

```diff
@@ -29,6 +29,13 @@ Each entry in the list shows the following details:
 - The most recent version of the image and the vulnerabilities for that version. Clicking the link for the base image opens [the image layer view](#image-layer-view).
 - The operating system and architecture of the image.
 - The base image and version used by the repository and the vulnerabilities for that version. Clicking the link for the base image opens [the image layer view](#image-layer-view).
+
+  > **Note**
+  >
+  > Docker Scout detects the base image of an image used in a repository by matching layer `diff_ids`.
+  > However, it's possible for different images and tags to use these layers. In these cases, Docker Scout's
+  > base image detection might not be precise and potentially return a different tag to the one used in the Dockerfile.
```

Comment:
Maybe it's just me, but my eyes stumbled a bit over parsing the scope of the _not_ when I first read this. The first half of the conjunction is essentially a subset of the second half, so we could just skip it?

```suggestion
  > base image detection could return a different tag to the one used in the Dockerfile.
```